packages/extension/src/routes/Home.vue

```diff
@@ -397,7 +395,7 @@ export default {
   computed: {
     ...mapState('ui', ['notifications', 'showNotifications', 'showSettings', 'theme', 'showDndMenu']),
     ...mapGetters('ui', ['sidebarInstructions', 'showReadyModal', 'dndMode']),
-    ...mapState('feed', ['showBookmarks', 'filter', 'sortBy', 'showFeed']),
+    ...mapState('feed', ['showBookmarks', 'filter', 'sortBy', 'showFeed', 'daFeedRef']),
```

Comment:
Why do you need the `daFeedRef` here from the state?
The `Home` component provides it to the state in the first place

Reply:
Oh, this is not actually being used, sorry for the misplacement!